ios/ImagePickerManager.m

```diff
@@ -30,6 +30,9 @@ @implementation ImagePickerManager
 NSString *errCameraUnavailable = @"camera_unavailable";
 NSString *errPermission = @"permission";
 NSString *errOthers = @"others";
+
+BOOL photoSelected = NO;
```

Comment:
I'd probably rename this to something a bit more clear like `isPickerOpen` ?

Reply:
no this is not isPickerOpen flag. Cause crash happens if we select photo twice while picker is open. So we just ignore all picks after first one

Reply:
Hmm you mean as in we double tap? I thought this issue was related to opening the picker twice.

Reply:
Yep, double tap in my case. Picker is not closing immediate after pick and user (as me) can tap once again